docs/Development Concepts/2.md

```diff
@@ -0,0 +1,70 @@
+
+### Waku Filter: `/vac/waku/filter/2.0.0-beta1`
+
+`WAKU2-FILTER` is a protocol that enables subscribing to messages that a peer receives.
+It enables a node to access the relay network without the connectivity and bandwidth requirements of relay nodes, but it comes with privacy drawbacks.
+Light nodes subscribe to full nodes and only receive the messages they desire.
+
+This is used to make fetching of a subset of messages more bandwidth preserving.
+
+### Waku Store: `/vac/waku/store/2.0.0-beta4`
+
+DApps running on a phone or in a browser are often offline:
+The browser could be closed or mobile app in the background.
+
+[Waku Relay](https://rfc.vac.dev/spec/11/) is a gossip protocol.
+As a user, it means that your peers forward you messages they just received.
+If you cannot be reached by your peers, then messages are not relayed;
+relay peers do **not** save messages for later.
+
+However, [Waku Store](https://rfc.vac.dev/spec/13/) peers do save messages they relay,
+allowing you to retrieve them at a later time.
+The Waku Store protocol is best-effort and does not guarantee data availability.
+Waku Relay should still be preferred when online;
+Waku Store can be used after resuming connectivity:
+For example, when the dApp starts.
+
+### Waku LightPush: `/vac/waku/lightpush/2.0.0-beta1`
+
+Waku Light Push enables a client to receive a confirmation when sending a message.
+
+The Waku Relay protocol sends messages to connected peers but does not provide any information on whether said peers have received messages.
+This can be an issue when facing potential connectivity issues.
+For example, when the connection drops easily, or it is connected to a small number of relay peers.
+
+Waku Light Push allows a client to get a response from a remote peer when sending a message.
+Note this only guarantees that the remote peer has received the message,
+it cannot guarantee propagation to the network.
+
+It also means weaker privacy properties as the remote peer knows the client is the originator of the message.
+Whereas with Waku Relay, a remote peer would not know whether the client created or forwarded the message.
+
+You can find Waku Light Push's specifications on [Vac RFC](https://rfc.vac.dev/spec/19/).
+
+`WAKU2-LIGHTPUSH` is a request/response protocol for this.
+
+## Additional Protocols
+
+This is in addition to protocols that specify messages, payloads, and recommended usages.
+Since these aren't negotiated libp2p protocols, they are referred to by their RFC ID.
+For example:
+
+- [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14)and [26/WAKU2-PAYLOAD](https://rfc.vac.dev/spec/26) for message payloads
+- [23/WAKU2-TOPICS](https://rfc.vac.dev/spec/23) and [27/WAKU2-PEERS](https://rfc.vac.dev/spec/27) for recommendations around usage
+
+There are also more experimental libp2p protocols such as:
+
+1. [`/vac/waku/swap/2.0.0-beta1`](https://rfc.vac.dev/spec/18/)
```

Comment:
```suggestion
```

No need to mention this one. Research is WIP and AFAIK will be replaced with service credentials.